packages/quantic/cypress/integration/document-suggestion/document-suggestion.cypress.ts

```diff
@@ -0,0 +1,195 @@
+import {configure} from '../../page-objects/configurator';
+import {DocumentSuggestionExpectations as Expect} from './document.suggestion.expectations';
+import {DocumentSuggestionActions as Actions} from './document-suggestion-actions';
+import {scope} from '../../reporters/detailed-collector';
+import {
+  interceptCaseAssist,
+  mockDocumentSuggestion,
+  interceptSuggestionIndefinitely,
+} from '../../page-objects/case-assist';
+import {sendRating} from '../../page-objects/actions/action-send-rating';
+
+interface DocumentSuggestionOptions {
+  maxDocuments: number;
+}
+
+describe('quantic-document-suggestion', () => {
+  const pageUrl = 's/quantic-document-suggestion';
+
+  const allDocuments = [
```

Comment:
There's currently an empty `cypress/fixtures` directory that could be used for large objects like this one.